Bottom: a6c71e7687d0f293426cffacba5b08fc4e487273
Top:    c5a9ce2d08e3d456c3d0ca15114901becddb0f4b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:39:01 +0000

Refresh of rename-default-silk-layers

---

diff --git a/src/create.c b/src/create.c
index 3089bdc..a7185df 100644
--- a/src/create.c
+++ b/src/create.c
@@ -224,8 +224,8 @@ CreateNewPCBPost (PCBType *pcb, int use_defaults)
 	return 1;
     }
 
-  pcb->Data->Layer[top_silk_layer].Name = strdup ("silk");
-  pcb->Data->Layer[bottom_silk_layer].Name = strdup ("silk");
+  pcb->Data->Layer[top_silk_layer].Name = strdup ("top silk");
+  pcb->Data->Layer[bottom_silk_layer].Name = strdup ("bottom silk");
 
   return 0;
 }
